### PR TITLE
usbguard: 0.7.4 -> 0.7.5

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

https://github.com/USBGuard/usbguard/releases/tag/usbguard-0.7.5

* no longer needs/uses qt as of 0.7.5
* asciidoctor -> asciidoc, drop pandoc
  (pandoc only seems to be used as part of doc spell-check
   which we don't enable anyway)
* docbook bits

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -16527,7 +16527,7 @@ in
 
   upower = callPackage ../os-specific/linux/upower { };
 
-  usbguard = libsForQt5.callPackage ../os-specific/linux/usbguard {
+  usbguard = callPackage ../os-specific/linux/usbguard {
     libgcrypt = null;
   };
```
Comment (jonringer) [inline pkgs/top-level/all-packages.nix]: you also need to kill the `usbguard-nox` entry on  pkgs/top-level/all-packages.nix:16535

Comment (dtzWill) [reply pkgs/top-level/all-packages.nix]: Good catch, thanks! Added alias to avoid breakage for anyone using `usbguard-nox`.

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
binaries seem to work
leaf package

```
[3 built, 86 copied (264.0 MiB), 51.9 MiB DL]
https://github.com/NixOS/nixpkgs/pull/68080
1 package were build:
usbguard
```

**Comment by ivan** [comment]
This is a somewhat unfortunate upgrade, because 0.7.4 had a useful and working GUI for allowing or denying a device, while 0.7.5 removed it entirely. https://github.com/USBGuard/usbguard/commit/bf3bb09edf491892617d35b55edec06f92d2743f says "the Qt applet will be maintained in a separate repository", but I do not see it in http://github.com/USBGuard.

**Pull request merged**